modules/context/api.go

```diff
@@ -303,6 +303,7 @@ func APIContexter() func(http.Handler) http.Handler {
 			ctx.Resp.Header().Set(`X-Frame-Options`, setting.CORSConfig.XFrameOptions)
 
 			ctx.Data["CsrfToken"] = html.EscapeString(ctx.csrf.GetToken())
+			ctx.Data["Context"] = &ctx
```

Comment:
do we realy need the context for api requests injected? (api do not have templates - but I guess we still use it in frontend somewhere ? :/ )

Reply:
Markdown render